src/components/Card/Card.tsx

```diff
@@ -0,0 +1,48 @@
+import styles from './Card.module.css';
+
+export default function Card({
+  numbers,
+  cardLogo,
+  expiration
+}: {
+  numbers: string[];
+  cardLogo: 'visa' | 'master' | '';
+  expiration: string[];
+}) {
+  return (
+    <div className={styles.card}>
+      <div className={styles.chipWrapper}>
+        <div className={styles.chip} />
+
+        {cardLogo === '' ? null : (
+          <img className={styles.cardLogo} src={cardLogo === 'visa' ? 'images/visa.jpg' : 'images/mastercard.jpg'} />
+        )}
```

Comment:
```suggestion
const CARD_LOGO_IMAGES = {
  visa: 'images/visa.jpg',
  master: 'images/mastercard.jpg'
};

        {cardLogo ? (
          <img className={styles.cardLogo} src={CARD_LOGO_IMAGES[cardLogo]} alt={`${cardLogo} 로고`}/>
        ) : null}
```
1. `cardLogo === ''` 조건식이 없어도 암묵적 타입 변환으로 삼항 연산자를 사용할 수 있을 것 같아요!(저는 이 방식을 더 선호하는 것 같습니다🙃)

2. 지금은 `src` 속성을 삼항 연산자로 설정하고 있는데 요구사항이 늘어났을 때를 대비해서 `CARD_LOGO_IMAGES`라는 객체에 카드 로고 src들을 관리하고 매핑해줘도 좋을 것 같습니다🫡

3. `alt` 속성 처리도 해줘도 좋을 것 같아요~